src/client/api/pythonApi.ts

```diff
@@ -208,15 +207,6 @@ export class LanguageServerProvider implements ILanguageServerProvider {
     }
 }
 
-@injectable()
-export class WindowsStoreInterpreter implements IWindowsStoreInterpreter {
-    constructor(@inject(IPythonApiProvider) private readonly apiProvider: IPythonApiProvider) {}
-
-    public isWindowsStoreInterpreter(pythonPath: string): Promise<boolean> {
-        return this.apiProvider.getApi().then((api) => api.isWindowsStoreInterpreter(pythonPath));
```

Comment:
Since we're no longer passing just `path:string`, we have the full env information, thus we don't need this.